### PR TITLE
Update prometheus

### PR DESCRIPTION
**Pull request by rodrmartinez**
- updates api version for ClusterRole and ClusterRoleBinding
- updates alert-rules format 
- updates kubernetes rules format
- updates  kube-state-metrics version
- updates prometheus version and args

### CODE REVIEW

**Comment by elieserr** [comment]
There is no `monitoring` namespace, maybe you can add a manifest for it.
Also, a small readme including a description and how to deploy the stack could be useful.

File: deploy/kubernetes/manifests-monitoring/prometheus-configmap.yaml [hunk 1]
```diff
@@ -13,111 +9,150 @@ data:
...
+        replacement: $1:$2
+      - action: labelmap
+        regex: __meta_kubernetes_service_label_(.+)
+      - source_labels: [__meta_kubernetes_namespace]
+        action: replace
+        target_label: kubernetes_namespace
+      - source_labels: [__meta_kubernetes_service_name]
+        action: replace
+        target_label: kubernetes_name
+    - job_name: 'kubernetes-services'
```
Comment (elieserr) [inline deploy/kubernetes/manifests-monitoring/prometheus-configmap.yaml]: this job is not reflected while looking at the targets in the UI

File: deploy/kubernetes/manifests-monitoring/prometheus-configmap.yaml [hunk 2]
```diff
@@ -13,111 +9,150 @@ data:
...
+        replacement: blackbox-exporter.example.com:9115
+      - source_labels: [__param_target]
+        target_label: instance
+      - action: labelmap
+        regex: __meta_kubernetes_service_label_(.+)
+      - source_labels: [__meta_kubernetes_namespace]
+        target_label: kubernetes_namespace
+      - source_labels: [__meta_kubernetes_service_name]
+        target_label: kubernetes_name
+    - job_name: 'kubernetes-ingresses'
```
Comment (elieserr) [inline deploy/kubernetes/manifests-monitoring/prometheus-configmap.yaml]: this is not reflected while looking at the targets in the UI

Comment (rodrmartinez) [reply deploy/kubernetes/manifests-monitoring/prometheus-configmap.yaml]: solved adding permissions in the CR, it is reflected in the UI only when an ingress is created

File: deploy/kubernetes/manifests-monitoring/prometheus-svc.yaml [hunk 3]
```diff
@@ -1,19 +0,0 @@
-apiVersion: v1
```
Comment (elieserr) [inline deploy/kubernetes/manifests-monitoring/prometheus-svc.yaml]: you completely removed this file

**Review by elieserr — CHANGES_REQUESTED** [review]

File: deploy/kubernetes/manifests-monitoring/prometheus-svc.yaml (hunk 3 above)
Comment (rodrmartinez) [reply deploy/kubernetes/manifests-monitoring/prometheus-svc.yaml]: I forgot to upload the migrated file,

**Comment by elieserr** [comment]
please rename the `.yml`  files to `.yaml`

**Review by elieserr — APPROVED** [review]
LGTM

**Pull request merged**